.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+
+pictures/
```

Comment:
This is half correct. Because what happened when if you pull your branch & don't know do you need this folder?

Something like this happened
![Capture](https://user-images.githubusercontent.com/5388080/126830111-9ec674eb-5b6a-44c8-8ace-7d45f219ca9b.PNG)

Right now we don't need photos but if you want to append is ok, my recommendation is to find a way to keep the folder but ignore the contented for git ... clues: `.gitkeeper`, exclude ignores.
This gonna help in the future when we need to store de data.